js/samples/azure-function-csharp/Program.cs

```diff
@@ -0,0 +1,16 @@
+using Microsoft.Extensions.Hosting;
```

Comment:
js/samples/azure-function-csharp/images/azid6.png

This image needs to be updated to select

_Cognitive Services **Immersive Reader** User_

This was a security fix

Reply:
done